### ISSUE TITLE
New Overworld Map

### ISSUE THREAD

**Issue by sedgwick** [IdleLands/Maps#1]
Mention was made of using the [Cobrahn Overworld](http://www.tantheus.com/Drakkar/Cobrahnmaps.html) for the next major map during the original release.

Information previously brought up regarding Cobrahn by @seiyria:
> For this map, the original way to get there was a portal in Norkos -7 (behind Slicer). Realistically, I think there should be more portals than that -- one in this dungeon in the same spot, as well as a few others in "deep" locations. Alternatively, it could just be a portal in Norkos but I think it should be harder to get to (although, since the game is random it should be more accessible, ie, in a few more dungeons).

Important information needed for this map:

- Determine which additional dungeons have access to Cobrahn, or a possible land route and not just teleports.
- Size? Looking at some of the other maps of the overworld, I could probably count out the exact size it was in the original game, but as a brand new overworld we can change the scale as needed. For reference, Norkos is a 250x250 map.
- Important Dungeons? Which additional maps should be included with the release of the overworld, and which should be added in later?
- And for a future date, are there any new mechanics that would be implemented? For example, something like the Fate Pools Dungeon has additional work that needs done outside of Tiled.
- Is the major cave system in the center of the map something that will be visible on the overworld, or will it be a separate dungeon file? Looking at the reference material, it would be separate. If it is separate, that is a massive section of the map that would just be filled with blocking mountain tiles (as best as I can tell), so something else could be made on top of the mountain.

**Comment by seiyria** [comment]
The only possible land route I could see would be a tunnel of sorts that takes you there. However, teleports would be a better idea IMO because then you don't need so many tunnels.

The world should be no larger than Norkos. In fact, I think it should be smaller. Norkos was an attempt to match the general size of the world exactly (since it was a tribute), but I don't think it's necessary going forward. I think taking the general ideas of the map and applying them in a similar way would be sufficient.

There are too many dungeons here to consider at launch, as it would be a large project and would take too long. If I were to prioritize, I would say that the "Timmy" dungeon, "Graagh" dungeon, and the "Undead City" dungeons are the most important. Feel free to personalize the maps as desired - we're not going for accuracy.

There will not be any new mechanics to the best of my knowledge. If anything strikes me during development or exploration of the maps, it can get added later. There's a lot of catching up to do before this becomes relevant, though.

All of the stuff on the overworld can just be part of the overworld. I like to minimize blank spaces as best as possible, so these things should be injected right in.

**Comment by sedgwick** [comment]
One of the dungeons off of Cabran could be a cemetery themed map with a dark stone dungeon crypt. Entrance overgrown with trees.

**Comment by seiyria** [comment]
There is what's called the Undead City, I think it can be made of tombstones and crypts.

**Comment by sedgwick** [comment]
Level range: current connections to cabran are on the astral plane and n-7. Getting there is something that will take some dedication and levels. Norkos content goes up to 200 with most of it under level 100, what should Cabran's easiest levels be, or high level stuff. Should it be geared for mostly high level content?

Classes: to help give the new map some extra content that would not be available on Norkos, possibly some of the other planned classes could be located here. Maybe also have extra Trainers for the Norkos hard to reach classes

**Comment by seiyria** [comment]
I think it should start at 50 and go to maybe 150. I think we can put 3 of the new classes there (I want magician to be in the funhouse yet), and then mirror some of the other classes so there are some norkos only ones too.

I think there might need to be other teleports, too, to various parts of cabran. Maybe in the deepest parts of some of the smaller dungeons (like mythos), and maybe one from idle island. that way it's not super uncommon to get there. each of these other teleports can go to a different region, too - idle island could go to the desert, n7 to the forest... maybe there are other fitting ways to do this, but that should illustrate the idea.

**Comment by sedgwick** [comment]
Add Barbarian Guardian in Cabran, flavorText can be how it has some loose connections and has a super-powered rage. That's how it wandered away from Norkos.

**Comment by sedgwick** [comment]
Ways to reach Cabran:
* The portal that requires Smasher/Slicer kills in N-7.
* Idle Cave, no current requirements.
* Astral Plane, requiring everything needed to reach the Astral Plane.
* Mythos, add -3. could be reached from -1, through a hole or rope in the Mythos Ruins. http://puu.sh/r48AO/d73a4865fd.png -3 could be a large city ruins, the rest of the city ruins on -1. Big maze with a few bosses (what kinds of bosses were never specified during our chat, anything in mind that would have survived whatever ruined the city, or possibly what has moved in since the they became ruined?) that are required to reach the Cabran portal.
* Homlet Caves, probably update this area and make it an old dwarf access to Cabran. require stuff from ancient dwarven ruins maps.
* Minotaur maze, requirements include all mino stuff. Probably open up the labyrinth to access the dirt region around it, have to walk around the entire outer section of labyrinth to access portal. http://puu.sh/r48WB/b73d344552.png

Since the requirements for each Cabran access are different, I feel some should go to different areas. Mythos ruined city could be very high level to fill in some of the 100-200 content, and lead to a strong region of Cabran, while Idle Cave might go to an easy location. Additionally Idle Cave access could be more difficult to be inline with the others.

**Comment by seiyria** [comment]
I figure Mythos could require all of the dragon kills.

**Comment by sedgwick** [comment]
Underwater dungeon. Water levels are everyone's favorite levels in games.

**Comment by sedgwick** [comment]
Time to get started.

**Comment by sedgwick** [comment]
Some small traces of the Norkos corruption may dot the landscape of Cabran, but overall the world was secluded enough that only a few references will appear. They should be part of a new lore storyline to keep the eventual storyline achievements easier to manage.

The 3 big bosses of Cabran will have control over the distinct geographical regions and require boss kills and collectibles from their regions to access them.

**Comment by sedgwick** [comment]
A good leveling trinket for Cabran might be nice.

**Comment by sedgwick** [comment]
Cabran to-do:

- [ ] Small overworld adjustments, since the world is mostly complete.
- [ ] Small dungeon adjustments
- [ ] Fill in collectibles
  - [ ] Names
  - [ ] flavorText
  - [ ] storylines
- [ ] Fill in bosses
  - [ ] flavorText
  - [ ] bossItems
- [ ] GM commands updated for Cabran.
- [ ] Treasure sets created.
- [ ] Norkos connections created.
- [ ] Create new items for custom-assets to release with the overworld.
- [ ] Remaining dungeons:
  - [ ] Undead City
  - [ ] Ironfist Castle -1
  - [ ] Ironfist Castle +2
  - [ ] Vanidor Lair
  - [ ] Lockers
  - [ ] Ancient Giant Keep
  - [ ] An underwater dungeon
- [ ] Place the remaining 3 trinkets

**Comment by sedgwick** [comment]
The "bit off more than I can chew" update:

My original plan was to make a complete overworld with all available dungeons, bosses, etc. ready upon initial release, but honestly that's way too large of an undertaking when making IdleLands updates is something I do in my spare time (what little remains). I'm only [human after all](https://www.youtube.com/watch?v=PXYeARRyDWk), so my new plan is to break up Cabran into many small pieces and release the maps over the coming weeks/months. Dungeons that have not been created yet will come much later down the line, once current maps are implemented and will have separate issues of their own in the future. Here is my new roadmap for Cabran release:

- [ ] Complete Cabran overworld
  - [ ] Collectibles (ready for collection)
  - [ ] Bosses (ready for combat)
  - [ ] Regions
  - [ ] Placeholder all dungeon stairs with signs
  - [ ] Placeholder teleports back to Norkos  
  - [ ] Open the Norkos Dungeon -7 access to Cabran, future connections will come as their dungeons release.
  - [ ] GM commands for overworld (during this time I'll review current commands, since I know a few are missing)
- [ ] Release Cabran overworld (certain areas will be inaccessible upon release, requiring future dungeons to gain access)
- [ ] Cabran overworld boss items
- [ ] Prioritize dungeons that provide access to regions on Cabran overworld (for example, the Tree Town access to the central Cabran mountain)
- [ ] Review and release Cabran connected dungeons
- [ ] Review and release dungeons branching from other dungeons
  - [ ] Remove placeholder teleports back to Norkos as intended access points release.

These are the features that will release a bit further down the line (as mentioned earlier, with issues of their own once the overworld issue is closed.

* Undead City
* Ironfist Castle -1 and Ironfist Castle +2
* Vanidor Lair
* Lockers
* Ancient Giant Keep
* An underwater dungeon
* Lich Tower, a very tall structure with the trainer at the top and plenty of traps.
* Place the remaining 3 trinkets
* Create new items for custom-assets.
* Other issues related to Cabran but already have issues of their own (for example, JesterQuest).

I'm guessing the initial overworld release will come within a month, with dungeons releasing much faster afterwards since they contain a lot fewer items to review. @seiyria, I'll also have to review certain map features that were created now that so much has changed in game since the maps first started being worked on, so I'll update you on that as the issues come back up.

**Comment by seiyria** [comment]
This is true. I like it in it's current state but I'm happy to review it as pieces come up and while most of it is probably fine, it's still nice to see it beforehand. I think your plan is probably not a bad idea as neither of us have quite the time we used to, haha.

Thanks for the update.
